packages/cli/src/transforms/toktx.ts

```diff
@@ -110,9 +112,11 @@ export const toktx = function (options: ETC1SOptions | UASTCOptions): Transform
 
 		let numCompressed = 0;
 
-		doc.getRoot()
-			.listTextures()
-			.forEach((texture, textureIndex) => {
+		await PromisePool
```

Comment:
Instead of using this PromisePool library, I'd use the `p-limit` library and do a more typical `listTextures().map`. Then the API surface is more familiar, Pushed a commit doing this https://github.com/spatialsys/glTF-Transform/commit/778914a9234d22425874551f32f10a4e1d93ec70

Reply:
thanks!